File: SDKs/Aspose.Tasks_Cloud_SDK_for_Java/src/main/java/com/aspose/tasks/model/AssignmentItems.java
package com.aspose.tasks.model;

import java.util.*;
import com.aspose.tasks.model.Link;
public class AssignmentItems {
  private List<AssignmentItem> List = new ArrayList<AssignmentItem>();
  private Link link = null;
  /**
	 * getList
	 * Gets List<AssignmentItem>
	 * @return List
	 */
  public List<AssignmentItem> getList() {
    return List;
  }

	/**
	 * setList
	 * Sets List<AssignmentItem>
	 * @param List List<AssignmentItem>
	 */
  public void setList(List<AssignmentItem> List) {
    this.List = List;
  }

  /**
	 * getLink
	 * Gets Link
	 * @return link
	 */
  public Link getLink() {
    return link;
  }

	/**
	 * setLink
	 * Sets Link
	 * @param link Link
	 */
  public void setLink(Link link) {
    this.link = link;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class AssignmentItems {\n");
    sb.append("  List: ").append(List).append("\n");
    sb.append("  link: ").append(link).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Tasks_Cloud_SDK_for_.NET/src/Com/Aspose/Tasks/Model/AssignmentItems.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Tasks.Model
{
    public class AssignmentItems
    {
        public List<AssignmentItem> AssignmentItem { get; set; }

        public Link link { get; set; }

        public override string ToString()
        {
            var sb = new StringBuilder();
            sb.Append("class AssignmentItems {\n");
            sb.Append("  List: ").Append(AssignmentItem).Append("\n");
            sb.Append("  link: ").Append(link).Append("\n");
            sb.Append("}\n");
            return sb.ToString();
        }
    }
}
